### PR TITLE
add fixturesDir configuration option

### PR DESCRIPTION
**Pull request by maciej-ka**
https://github.com/react-cosmos/react-cosmos/issues/287
Enables to change fixtures directory name in configuration, default value is `__fixtures__`.

### CODE REVIEW

**Comment by skidding** [comment]
@maciej-ka thank you, this looks good!

I wonder why didn't Travis got triggered on this PR. Looking at #353 and #332 it seems like it should work out outside PRs already. @ganderzz @jnishiyama did you do anything extra to ensure Travis ran on your fork PRs?

**Comment by jnishiyama** [comment]
@skidding Naw, Didn't do anything extra. Why don't you or someone else try making the PR to attempt to isolate the issue?

**Comment by skidding** [comment]
@maciej-ka how about adding a unit test for this? Tests for `react-cosmos-voyager` have a twist to them but are then very fun to write.

[Here](https://github.com/react-cosmos/react-cosmos/blob/10402ea8afdf626713f993fbcd1832649678e091/packages/react-cosmos-voyager/scripts/generate-use-cases.js#L6-L65) are the use cases. After you add another use case to that map, you go to `packages/react-cosmos-voyager` and run `yarn generate-use-cases`. This will write the corresponding files to the disk. Then in the tests file you add a use case to match those files. I'd start from a simple example like [relative-fixtures](https://github.com/react-cosmos/react-cosmos/blob/10402ea8afdf626713f993fbcd1832649678e091/packages/react-cosmos-voyager/src/__tests__/index.js#L47-L69).

**Comment by maciej-ka** [comment]
I was thinking about adding them too, big thanks for instructions

File: packages/react-cosmos-voyager/scripts/generate-use-cases.js [hunk 1]
```diff
@@ -62,6 +62,16 @@ const useCases = {
     pkgs/nested/Baz/src/index.jsx
     pkgs/nested/Baz/fixtures/blank.js
   `),
+  'fixtures-dir-setting': (`
+    components/Foo.js
+    components/nested/Bar.jsx
+    components/nested/Baz.jsx
+    components/__fixtures__/Foo/blank.js
```
Comment (skidding) [inline packages/react-cosmos-voyager/scripts/generate-use-cases.js]: Shouldn't this be `components/non-default-fixtures-dir/Foo/blank.js`

**Comment by ganderzz** [comment]
@skidding Late to the party, but no, didn't have to do anything to trigger Travis. A newer PR has a build, so seems to be isolated to this PR..?

File: packages/react-cosmos-voyager/scripts/generate-use-cases.js (hunk 1 above)
Comment (maciej-ka) [inline packages/react-cosmos-voyager/scripts/generate-use-cases.js]: Right, should be fixed now. I changed generated folder and forgot to update here.

**Comment by skidding** [comment]
This is great. Will try it out locally and merge it this weekend.

@maciej-ka maybe you need to have a Travis account (login via Github) in order for Travis to trigger builds when you open a PR.

**Comment by skidding** [comment]
Beautiful! ✨

**Pull request merged**